GNNLux/src/layers/conv.jl

```diff
@@ -628,3 +628,53 @@ function Base.show(io::IO, l::GINConv)
     print(io, ", $(l.ϵ)")
     print(io, ")")
 end
+
+@concrete struct NNConv <: GNNContainerLayer{(:nn,)}
+    nn <: AbstractExplicitLayer    
+    aggr
+    in_dims::Int
+    out_dims::Int
+    use_bias::Bool
+    add_self_loops::Bool
+    use_edge_weight::Bool
+    init_weight
+    init_bias
+    σ
+end
+
+
+function NNConv(ch::Pair{Int, Int}, nn, σ = identity; 
+                aggr = +, 
+                init_bias = zeros32,
+                use_bias::Bool = true,
+                init_weight = glorot_uniform,
+                add_self_loops::Bool = true,
+                use_edge_weight::Bool = false,
+                allow_fast_activation::Bool = true)
+    in_dims, out_dims = ch
+    σ = allow_fast_activation ? NNlib.fast_act(σ) : σ
+    return NNConv(nn, aggr, in_dims, out_dims, use_bias, add_self_loops, use_edge_weight, init_weight, init_bias, σ)
+end
+
+function (l::NNConv)(g, x, edge_weight, ps, st)
+    nn = StatefulLuxLayer{true}(l.nn, ps, st)
+
+    # what would be the order of args here?
+    m = (; nn, l.aggr, ps.weight, bias = _getbias(ps), 
+        l.add_self_loops, l.use_edge_weight, l.σ)
```

Comment:
any suggestions?

Reply:
the order of fields in a named tuple does not matter